proj/src/main/webapp/static/chat.html

```diff
@@ -232,5 +247,6 @@ <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
     </div>
     <!-- end of body-row -->
     <script src="../bundle.js"></script>
+    <script src="../chat.js"></script>
```

Comment:
This shouldn't be here. This might have been what was causing your errors.

Reply:
yup it was!